package/opt/hassbian/suites/mssql.sh

```diff
@@ -25,8 +25,8 @@ echo "Changing to Home Assistant venv"
 source /srv/homeassistant/bin/activate
 
 echo "Installing dependencies for MS SQL"
-pip3 install --upgrade setuptools wheel
-pip3 install pymssql
+python -m pip install --upgrade setuptools wheel Cython
```

Comment:
Capital C on Cython shouldn't be capitalized.
Reason for this being a problem is that there's no "pipwheels" for this version of Python yet. Hopefully this will be corrected when Buster is released and Raspbian rebases on Buster but until then this is the best option.

Reply:
Capital C is the official instruction <https://pypi.org/project/Cython/> ?

Reply:
Then lets follow them :)